po/es.po

```diff
@@ -6099,181 +6199,182 @@ msgstr "controles"
 
 #: ../src/iop/colorbalance.c:927
 msgid "optimize luma from patches"
-msgstr "optimizar luminosidad desde los parches"
+msgstr "optimizar la luminosidad a partir de parches"
 
-#: ../src/iop/colorbalance.c:929 ../src/iop/colorbalance.c:2609
+#: ../src/iop/colorbalance.c:929 ../src/iop/colorbalance.c:2601
 msgid "optimize luma"
 msgstr "optimizar luminosidad"
 
 #: ../src/iop/colorbalance.c:933
 msgid "neutralize colors from patches"
-msgstr "neutralizar los colores de los parches"
+msgstr "neutraliza los colores desde los parches"
 
-#: ../src/iop/colorbalance.c:935 ../src/iop/colorbalance.c:2618
+#: ../src/iop/colorbalance.c:935 ../src/iop/colorbalance.c:2610
 msgid "neutralize colors"
-msgstr "neutralizar los colores"
+msgstr "neutraliza los colores"
 
-#: ../src/iop/colorbalance.c:2324
+#: ../src/iop/colorbalance.c:2316
 msgid "lift, gamma, gain (ProPhotoRGB)"
-msgstr "realce, gamma, ganancia (ProPhotoRGB)"
+msgstr "elevación, gamma, ganancia (ProPhotoRGB)"
```

Comment:
Even though I'm Spanish I have no idea about photography terminology, so I'm not sure about the "lift" -> "elevación" translation, which sounds like a direct translation, but in my mind, "realce" sounds better

I'm not asking for a change here (and everywhere else this appears) but for confirmation that this is indeed the right translation.